src/list_devices.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python3
```

Comment:
Please use env shebangs for all Python code, this allows non-system interpreters to be used.

We should probably s/python3/python/ as well and fix it in the configure process for distributions that ships Python 2 and 3 simultaneously.

Reply:
Can you explain second part?

Reply:
i.e. use `#!/usr/bin/env python` instead of `#!/usr/bin/env python3`, treat Debian-like distros as special case.

Reply:
Last time I've checked on ubuntu `python` invoked 2.7 and `python3` 3.x. On arch `python` and `python3` invoke python 3.x.
I've written this script on python 3 and it won't work on 2.7

Reply:
Yes, however not all distros follow the naming fashion(i.e. Archlinux), we should leave the interpreter naming to the configure/installation program and don't assume certain interpreter name.

Reply:
done